### ISSUE TITLE
oidc-agent 5.0.0 segfaults

### ISSUE THREAD

**Issue by pllopis** [indigo-dc/homebrew-oidc-agent#26]
I was on 4.5.1 which was working fine. After upgrading to 5.0.0 it started segfaulting.

```
$ oidc-agent
zsh: segmentation fault  oidc-agent
llopis ~  $ lldb --file $(which oidc-agent)
(lldb) target create "/opt/homebrew/bin/oidc-agent"
Current executable set to '/opt/homebrew/bin/oidc-agent' (arm64).
(lldb) r
Process 8884 launched: '/opt/homebrew/bin/oidc-agent' (arm64)
Process 8884 exited with status = 45 (0x0000002d)
(lldb)
```

I'm using an M1 macbook on 13.2.1 (22D68)

**Comment by zachmann** [comment]
Thanks for the info, and I'm sorry for the inconvenience. 
We just released 5.0.1 which fixes the segfault.

**Comment by pllopis** [comment]
Many thanks for the speedy fix! I can confirm that 5.0.1 fixes the issue.
